src/main/java/com/sngular/kloadgen/property/editor/NameStrategyPropertyEditor.java

```diff
@@ -55,13 +55,21 @@ private void init() {
     nameStrategyComboBox.addActionListener(this);
   }
 
-  private void fillSerializer(final JComboBox<String> objectJComboBox) {
+    private void fillSerializer(final JComboBox<String> objectJComboBox) {
+    objectJComboBox.addItem("io.apicurio.registry.serde.avro.strategy.RecordIdStrategy");
```

Comment:
I think this could be refactored to have an Util class to have all the naming strategies instead of use literals in it